### ISSUE TITLE
Doesn't Return Bot access Token

### ISSUE THREAD

**Issue by Fyrebox** [mjpearson/passport-slack#13]
As per the slack API documentation ([https://api.slack.com/docs/oauth](https://api.slack.com/docs/oauth) see section "Bot user access tokens"), when a bot created, the JSON payload should return the access token along with the bot user access token. However, we only get the accessToken. Is there a way to get both tokens by adding a parameter to the Oauth flow?

**Comment by franz-josef-kaiser** [comment]
Any update on this?

**Comment by Fyrebox** [comment]
Nope, I gave trying to make it work. I decided to use the Slack button to start the OAuth flow and get the tokens with a simple get request

**Comment by amackera** [comment]
I would love this as well! I'm confused about how bot tokens fit into the oauth flow.

**Comment by amackera** [comment]
If somebody else comes across this issue, simply include `params` in the `verify` callback:
```javascript
passport.use(new SlackStrategy({
  clientID: process.env.SLACK_CLIENT_ID,
  clientSecret: process.env.SLACK_CLIENT_SECRET,
  scope: 'incoming-webhook commands bot users:read',
}, (accessToken, refreshToken, params, profile, done) => {
   console.log(params.bot); // Should have `bot_user_id` and `bot_access_token`
}));
```
It's probably documented elsewhere --- took me forever to figure that out, sadly.
